src/styles/components/_product-media.scss

```diff
@@ -1,30 +1,86 @@
 .product-media {
-  display: flex;
+  @include flex-center();
   flex-direction: column;
-  align-items: center;
   box-sizing: border-box;
+  height: 450px;
   padding: rem(16px);
+  border-radius: 15%;
 
-  &__header {
-    margin-bottom: rem(24px);
+  @include respond-above('sm') {
+    height: 600px;
   }
 
-  &__title {
-    margin: 0;
+  @include respond-above('md') {
+    height: 430px;
+
+    border-radius: 15%;
+
+    background: $light;
+  }
+
+  @include respond-above('lg') {
+    height: 560px;
+  }
+
+  @include respond-above('xl') {
+    height: 650px;
   }
 
   &__image-container {
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    width: rem(200px);
-    height: rem(200px);
-    background-color: black;
+    @include flex-center();
+
+    height: 500px;
+
+    @include respond-above('md') {
+      height: 320px;
+    }
+
+    @include respond-above('lg') {
+      height: 420px;
+    }
+
+    @include respond-above('xl') {
+      height: 500px;
+    }
   }
 
   &__image {
-    width: 60%;
-    height: 60%;
-    background-color: blueviolet;
+    display: block;
+    max-width: 90%;
+    max-height: 100%;
+  }
+
+  &__image.smooth {
+    @include transition($what: opacity, $time: 0.2s, $how: cubic-bezier(0.3, 0, 0.4, 1));
+
+    opacity: 0.5;
+  }
+
+  &__image.smooth.loaded {
+    opacity: 1;
+  }
+
+  &__button {
+    width: rem(80px);
+    height: rem(80px);
+    border: none;
+    border-radius: 10%;
+    margin-right: rem(5px);
```

Comment:
See this comment [comment](https://github.com/Cobda/cobda-web/pull/56/files#r619648306)

Reply:
I can't see the comment. Please mention it again.